rename.rb

```diff
@@ -63,6 +63,7 @@ def usage
 FileUtils.rm_rf(old_dirs)
 
 FileUtils.mv('README.plugin.md', 'README.md')
+FileUtils.mv('ci.yml.tpl', 'ci.yml')
```

Comment:
This fails for me:
```console
$ ./rename.rb foreman_best_plugin
/usr/share/ruby/fileutils.rb:1181:in `rename': No such file or directory @ rb_file_s_rename - (ci.yml.tpl, ci.yml) (Errno::ENOENT)
	from /usr/share/ruby/fileutils.rb:1181:in `block in mv'
	from /usr/share/ruby/fileutils.rb:2481:in `block in fu_each_src_dest'
	from /usr/share/ruby/fileutils.rb:2497:in `fu_each_src_dest0'
	from /usr/share/ruby/fileutils.rb:2479:in `fu_each_src_dest'
	from /usr/share/ruby/fileutils.rb:1172:in `mv'
	from ./rename.rb:66:in `<main>'
```

Reply:
You should use `Find.prune` if the directory is `.git`. https://docs.ruby-lang.org/en/master/Find.html documents it.